docs/products/access/topics/integration-extension/hooks/customize-token/customize-token-hook.md

```diff
@@ -0,0 +1,44 @@
+# Customize Token Web Hook
+
+The customize token web hook is meant allow access control manipulation. For security reasons, access escalation is not allowed and the
+implementor can only reduce the scopes that are associated with the resulting tokens.
+
+## Web Hook specification
+
+The web hook implementor receives multiple attributes, such as:
```

Comment:
It may be unclear for the customer who is the "web hook implementor". Would be nice to mention who is he, where the hook should be implemented and how (API endpoint). An example implementation is also a benefit that makes it more clear.

Reply:
That's not something that is specific to this hook and the answer may vary depending on the customer and agreement. I can easily imagine us being responsible for implementing the hook at a times.